src/main/scala/firrtl/stage/Forms.scala

```diff
@@ -13,84 +14,84 @@ import firrtl.stage.TransformManager.TransformDependency
 
 object Forms {
 
-  val ChirrtlForm: Seq[TransformDependency] = Seq.empty
-
-  val MinimalHighForm: Seq[TransformDependency] = ChirrtlForm ++
-    Seq( classOf[passes.CheckChirrtl],
-         classOf[passes.CInferTypes],
-         classOf[passes.CInferMDir],
-         classOf[passes.RemoveCHIRRTL] )
-
-  val WorkingIR: Seq[TransformDependency] = MinimalHighForm :+ classOf[passes.ToWorkingIR]
-
-  val Resolved: Seq[TransformDependency] = WorkingIR ++
-    Seq( classOf[passes.CheckHighForm],
-         classOf[passes.ResolveKinds],
-         classOf[passes.InferTypes],
-         classOf[passes.CheckTypes],
-         classOf[passes.Uniquify],
-         classOf[passes.ResolveFlows],
-         classOf[passes.CheckFlows],
-         classOf[passes.InferBinaryPoints],
-         classOf[passes.TrimIntervals],
-         classOf[passes.InferWidths],
-         classOf[passes.CheckWidths],
-         classOf[firrtl.transforms.InferResets] )
-
-  val Deduped: Seq[TransformDependency] = Resolved :+ classOf[firrtl.transforms.DedupModules]
-
-  val HighForm: Seq[TransformDependency] = ChirrtlForm ++
+  def ChirrtlForm: Seq[TransformDependency] = Seq.empty
```

Comment:
Initialization order.  You're right, though,`lazy val` would be strictly better than `def` for efficiency here -- I put `def` because I was lazy.

I'll change those.